slice_test.go

```diff
@@ -20,3 +22,89 @@ func ExampleUniqueInterfaces() {
 	fmt.Println(u.UniqueInterfaces([]interface{}{13, "foo", "bar", 42, 13, 43, "baz"}))
 	// Output: [13 foo bar 42 43 baz]
 }
+
+func BenchmarkUniqueStrings(b *testing.B) {
+	cases := []struct {
+		Name string
+		Data []string
+	}{
+		{"slice1", []string{strings.Repeat("foo", 2), strings.Repeat("bar", 2), "foo", strings.Repeat("baz", 2), "foo", "bar", "foobar", "baz", "foobaz"}},
+		{"slice2", []string{strings.Repeat("foo", 3), strings.Repeat("bar", 3), "foo", strings.Repeat("baz", 3), "foo", "bar", "foobar", "baz", "foobaz"}},
```

Comment:
this second benchmark is useless, it could be interesting to test something very big instead